competitions/template/ansible/inv/prod/hosts

```diff
@@ -0,0 +1,5 @@
+[mediawiki]
+torque.leverforchange.org ansible_ssh_user="{{ lookup('env','OTSUSER') }}"
```

Comment:
It's `OTS_USERNAME` now.

(And, yeah, I know, we *could* have put an underscore in "USERNAME", but colloquially it's become one word, like "password", "filesystem", and (more controversially) "website".)

Reply:
I find nothing controversial about website :)